src/firefly/js/ui/CatalogSearchMethodType.jsx

```diff
@@ -0,0 +1,262 @@
+                                 initialState={{
+                                          fieldKey: 'axialratio',
+                                          value: '.26',
+                                          validator: Validate.floatRange.bind(null, 0, 1, 0,'Axial Ratio'),
+                                          tooltip: 'Enter the Axial ratio of the search e.g - 0.26',
+                                          label : 'Axial Ratio:',
+                                          labelWidth : 100
+                                      }}/>
+            </div>
+        );
+    } else if (searchType === SpatialMethod.Box.value) {
+
+        return (
+            <div className='size'>
+                {radiusInField('Side:', 'Enter side size of the box search', 1 / 3600, 7200 / 3600)}
+            </div>
+
+        );
+    } else if (searchType === SpatialMethod.get('Multi-object').value) {
+        return (
+            <div style={{padding:5, display:'flex', flexDirection:'column', flexWrap:'no-wrap', alignItems:'center' }}>
+                <FileUpload
+                    wrapperStyle={{margin: '5px 0'}}
+                    fieldKey='fileUpload'
+                    initialState={{
+                        tooltip: 'Select a  file to upload',
+                        label: 'Filename:'}}
+                />
+                {radiusInField()}
+            </div>
+        );
+    } else if (searchType === SpatialMethod.Polygon.value) {
+
+        return (
+            <div style={{padding:5, display:'flex', flexDirection:'column', flexWrap:'no-wrap', alignItems:'center' }}>
+                <InputAreaFieldConnected fieldKey='polygoncoords'
+                                wrapperStyle={{display:'flex', flexDirection:'row', alignItems:'center' }}
+                                initialState={{
+                                               tooltip:'Enter polygon coordinates search'
+                                            }}
+                                label='Coordinates:'
+                                tooltip='Enter polygon coordinates search'
+                />
+                <ul>
+                    <li>- Each vertex is defined by a J2000 RA and Dec position pair</li>
+                    <li>- A max of 15 and min of 3 vertices is allowed</li>
+                    <li>- Vertices must be separated by a comma (,)</li>
+                    <li>- Example: 20.7 21.5, 20.5 20.5, 21.5 20.5, 21.5 21.5</li>
+                </ul>
+            </div>
+        );
+
+    } else {
+        return (
+
+            <div className='size'>
+                Search the catalog with no spacial constraints
+            </div>
+        );
+    }
+}
+
+function renderTargetPanel(groupKey, searchType) {
+    const visible = (searchType === SpatialMethod.Cone.value || searchType === SpatialMethod.Box.value || searchType === SpatialMethod.Elliptical.value) && searchType != SpatialMethod.get('Multi-Object').value;
```

Comment:
I think '&& searchType != SpatialMethod.get('Multi-Object').value' is not necessary.
Also, there might be another way to do these 'in' comparison that's easier to read.